classes/opponentPhaseTEMPLATE.lua

```diff
@@ -4,13 +4,24 @@ local opponentPhaseTEMPLATE = {}
 opponentPhaseTEMPLATE.__index = opponentPhaseTEMPLATE
 
 
+-- Uncomment and override any of these if needed
+--opponentPhaseTEMPLATE.STUN_LENGTH = 0.3
+--opponentPhaseTEMPLATE.IFRAME_LENGTH = 0.7
+--opponentPhaseTEMPLATE.RADIUS = 25
+--opponentPhaseTEMPLATE.SPAWN_X = (ARENA_WIDTH / 2)
+--opponentPhaseTEMPLATE.SPAWN_Y = (ARENA_HEIGHT / 2)
+--opponentPhaseTEMPLATE.SPAWN_ANG = (math.pi / 2)
+--opponentPhaseTEMPLATE.NUM_LIVES = 3
+--opponentPhaseTEMPLATE.DEFAULT_BULLET_COOLDOWN = nil 	-- nil means don't use default shooting method
+--opponentPhaseTEMPLATE.DEFAULT_BULLET_SPEED = nil 	-- nil means use default speed
```

Comment:
Here are the list of properties you can override!! Copy any of these lines of code and uncomment, it will use this property instead of the default. `DEFAULT_BULLET_COOLDOWN` is the one that enabled the auto-shooting mechanic. Add that one line of code to enable simple shooting